packages/next/src/client/components/react-dev-overlay/ui/components/errors/call-stack/call-stack.stories.tsx

```diff
@@ -62,6 +62,7 @@ export const MultipleFrames: Story = {
           lineNumber: 5,
         },
       },
+      ...Array(5).fill(ignoredFrame),
```

Comment:
Added this to show a more realistic stacktrace where ignored and not-ignored frames are interleaved.